connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceConnectorTest.java

```diff
@@ -261,6 +302,54 @@ public void testNewTopicConfigs() throws Exception {
         verify(connector).createNewTopics(any(), any());
     }
 
+    @Test
+    public void testIncrementalAlterConfigsRequested() throws Exception {
+        MockAdminClient admin = spy(new MockAdminClient());
+        MirrorSourceConnector connector = spy(new MirrorSourceConnector(new SourceAndTarget("source", "target"),
+                new DefaultReplicationPolicy(), MirrorSourceConfig.USE_INCREMENTAL_ALTER_CONFIG_DEFAULT, new DefaultConfigPropertyFilter(), admin));
+        final String topic = "testtopic";
+        List<ConfigEntry> entries = new ArrayList<>();
+        entries.add(new ConfigEntry("name-1", "value-1"));
+        Config config = new Config(entries);
+        doReturn(Collections.singletonMap(topic, config)).when(connector).describeTopicConfigs(any());
+        doReturn(alterConfigsResult(new ConfigResource(ConfigResource.Type.TOPIC, topic), new UnsupportedVersionException("Unsupported API"))).when(admin).incrementalAlterConfigs(any());
+        doNothing().when(connector).alterConfigs(any());
+        connector.syncTopicConfigs();
+        verify(connector).syncTopicConfigs();
+        verify(connector).incrementalAlterConfigs(any());
+        verify(connector, times(1)).alterConfigs(any());
+    }
+
+    @Test
+    public void testIncrementalAlterConfigsRequired() throws Exception {
```

Comment:
Can we add a test that causes `incrementalAlterConfig()` to emit a `OpType.DELETE` op?